src/test/components/elements/Dashboard.test.tsx

```diff
@@ -0,0 +1,105 @@
+import { mockNewsPods, mockPromoPods } from "../../../mocks/data";
+import { IsaacPodDTO, UserRole } from "../../../IsaacApiTypes";
+import { Dashboard } from "../../../app/components/elements/Dashboard";
+import { renderTestEnvironment } from "../../utils";
+import { screen, waitFor } from "@testing-library/react";
+
+const mockPromoItem = mockPromoPods.results[0];
+const mockFeaturedNewsItem = mockNewsPods.results[1];
+
+describe("Dashboard", () => {
+  const setupTest = (role: UserRole | "ANONYMOUS", props = {}) => {
+    renderTestEnvironment({
+      role: role,
+      PageComponent: Dashboard,
+      componentProps: {
+        featuredNewsItem: mockFeaturedNewsItem as IsaacPodDTO,
```

Comment:
I don't think these castings are needed (the same goes for the other one in `PromoContent.test.tsx`). I'll approve the PR anyway, since it's quite minor.

Reply:
I thought it was required since the component was expecting props as a certain type, but it seems to work without  🎉

Reply:
I think that as long as the object has the expected shape, it does not need to implement an interface / type. Duck typing.